packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
 
 <img src="./img/two-level.png" alt="Example of grouped vertical navigation" width="634"/>
 
-**Expandable vertical navigation**
+#### Expandable two-level navigation
 
 When you have a large number of secondary navigation items, you can use an expandable navigation to collapse and expand options as needed.
 
 <img src="./img/expandable-nav.png" alt="Example of expandable vertical navigation" width="634"/>
 
+#### Expandable three-level navigation
 
+When you have three levels of navigation items, you can use a three-level expandable navigation to expose tertiary navigation items. Expansion works well when there are small numbers of items at the current level and the entire page hierarchy can be visualized at the same time.
+
+<img src="./img/expandable-nav-tertiary.png" alt="Example of expandable tertiary vertical navigation" width="634"/>
+
+#### Fly-out navigation
+
+Fly-out navigation exposes navigation items nested within a parent node in an overlay panel that appears to the right of the parent item on hover or click. Fly-out menus can be used to expose secondary or tertiary levels of navigation. While fly-out menus can be cascaded, this is not recommended as the mouse interaction required to operate cascading menus can be difficult for some users.
+
+<img src="./img/fly-out.png" alt="Example of fly-out navigation" width="917"/>
+
+The advantage of fly-out menus is that they allow a user to easily scan through menu items in a deep page hierarchy. But keep in mind that the page in view may not be exposed as a selected menu item at the top level. Therefore, when using fly-outs, we strongly recommend use of [breadcrumbs](/components/breadcrumb) to help users understand where they are currently working within the site hierarchy.
+
+Also, consider how likely it is that your application will be used on a mobile phone. Fly-outs are not mobile friendly and supporting unique  menu patterns on different platforms (mobile vs desktop).
+
+#### Drill-down navigation
+
+Drill-down menus replace the current navigation menu with the next set of child items in the page hierarchy when the user clicks on a parent item to drill down. A back link is provided at the top of the menu to return to the parent level.
+
+<img src="./img/drill-down.png" alt="Example of drill-down navigation" width="916"/>
+
+Using drill-down navigation, only the navigation items for the current page and its siblings will be visible at any given time. This pattern is very mobile-friendly as it minimizes the need for scrolling to see the entire menu. However, it should be avoided if you expect users to frequently move between levels.
+
+### Combining vertical navigation patterns
+
+PatternFly's vertical menus are designed to be composable. This means that you can combine different patterns within the same navigation system to address the needs of users working at different levels in the page hierarchy.
+
+When combining different types of menus, here are some guidelines:
```

Comment:
I don't think the "here are some guidelines" is needed here, can just leave it at "When combining different types of menus: "